ddtrace/llmobs/_integrations/utils.py

```diff
@@ -396,6 +396,192 @@ def openai_set_meta_tags_from_chat(span: Span, kwargs: Dict[str, Any], messages:
+                processed_item["content"] = str(processed_item_content)
+                processed_item["role"] = item["role"]
+        elif "call_id" in item and "arguments" in item:
+            # Process `ResponseFunctionToolCallParam` type from input messages
+            try:
+                arguments = json.loads(item["arguments"])
+            except json.JSONDecodeError:
+                arguments = {"value": str(item["arguments"])}
+            processed_item["tool_calls"] = [
+                {
+                    "tool_id": item["call_id"],
+                    "arguments": arguments,
+                    "name": item.get("name", ""),
+                    "type": item.get("type", "function_call"),
+                }
+            ]
+        elif "call_id" in item and "output" in item:
+            # Process `FunctionCallOutput` type from input messages
+            output = item["output"]
+
+            if isinstance(output, str):
+                try:
+                    output = json.loads(output)
+                except json.JSONDecodeError:
+                    output = {"output": str(output)}
+            processed_item.update(
+                {
+                    "role": "tool",
+                    "content": output,
+                    "tool_id": item["call_id"],
+                }
+            )
+        if processed_item:
+            processed.append(processed_item)
+
+    return processed
+
+
+def openai_get_output_messages_from_response(response: Optional[Any]) -> List[Dict[str, Any]]:
+    """
+    Parses the output to openai responses api into a list of output messages
+
+    Args:
+        response: An OpenAI response object containing output messages
+
+    Returns:
+        - A list of processed messages
+    """
+    if not response or not getattr(response, "output", None):
+        return []
+
+    messages: List[Any] = response.output
+    processed: List[Dict[str, Any]] = []
+
+    for item in messages:
+        message = {}
+        message_type = getattr(item, "type", "")
+
+        if message_type == "message":
+            text = ""
+            for content in getattr(item, "content", []):
+                text += getattr(content, "text", "") or ""
+                text += getattr(content, "refusal", "") or ""
+            message.update({"role": getattr(item, "role", "assistant"), "content": text})
+        elif message_type == "reasoning":
+            message.update(
+                {
+                    "role": "reasoning",
+                    "content": safe_json(
+                        {
+                            "summary": getattr(item, "summary", ""),
+                            "encrypted_content": getattr(item, "encrypted_content", ""),
+                            "id": getattr(item, "id", ""),
+                        }
+                    ),
+                }
+            )
+        elif message_type == "function_call":
+            arguments = getattr(item, "arguments", None)
```

Comment:
```suggestion
            arguments = getattr(item, "arguments", "{}")
```
Looks like typing is off. Can we try this?